tests/network/hotplug_bridge.go

```diff
@@ -339,10 +380,28 @@ func removeInterface(vm *v1.VirtualMachine, name string) error {
 }
 
 func verifyBridgeDynamicInterfaceChange(vmi *v1.VirtualMachineInstance, plugMethod hotplugMethod) *v1.VirtualMachineInstance {
+	virtClient := kubevirt.Client()
 	if plugMethod == migrationBased {
-		migration := libmigration.New(vmi.Name, vmi.Namespace)
-		migrationUID := libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(kubevirt.Client(), migration)
-		libmigration.ConfirmVMIPostMigration(kubevirt.Client(), vmi, migrationUID)
+		By("Waiting for MigrationRequired condition to appear")
+		Eventually(matcher.ThisVMI(vmi), 1*time.Minute, 2*time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceMigrationRequired))
+
+		By("Ensuring live-migration started")
+		var migration *v1.VirtualMachineInstanceMigration
```

Comment:
I would prefer to check the migration started on the VMI itself and not the VMIM. Is that possible and makes sense?

Reply:
The test still requires the VMIM (even before this change) in order to track the migration completion.

Reply:
Hi, In general it makes sense, I will have to test it in a follow up.

The change proposed in this commit was meant to fetch the VMI in order to make sure it actually ended.